lib/mongo_ecto.ex

```diff
@@ -537,13 +541,13 @@ defmodule Mongo.Ecto do
   end
 
   @doc false
-  def insert(_repo, meta, _params, [_|_] = returning, _opts) do
+  def insert(_repo, meta, _params, _on_conflict, [_|_] = returning, _opts) do
     raise ArgumentError,
       "MongoDB adapter does not support :read_after_writes in models. " <>
       "The following fields in #{inspect meta.schema} are tagged as such: #{inspect returning}"
   end
 
-  def insert(repo, meta, params, [], opts) do
+  def insert(repo, meta, params, _, [], opts) do
```

Comment:
Its just ignore, I don't know the behavior of mongodb in that case, we can discuss that...